tests/Microsoft.Graph.DotnetCore.Core.Test/Serialization/SerializerTests.cs

```diff
@@ -404,5 +404,56 @@ public void SerializeDeserializeJson()
             Assert.Equal(43, jsArr[2]);
             Assert.Equal(false, jsArr[3]);
         }
+        
+        [Fact]
```

Comment:
[nit] global comment for the three tests:

Consider using data attributes to separate test cases. If the serializer fails for a single type, that will generate an individual test case report.

An example setup can be found here: https://andrewlock.net/creating-parameterised-tests-in-xunit-with-inlinedata-classdata-and-memberdata/

Reply:
Fixed with edd6f1cf